src/products/mozilla_org/testing.md

```diff
@@ -58,65 +58,67 @@ Updated translations are pushed to the production server almost daily.
+* Click on the **Source code** link, you will be directed to the topic on [MDN](https://developer.mozilla.org/docs/Mozilla/Developer_guide/Source_Code) site in your language if it is localized.
+* The [Firefox Privacy Notice](https://www.mozilla.org/privacy/firefox/) document is localized in limited number of languages.
+* Click on the **Need Help** link, you will be directed to the [SUMO](https://support.mozilla.org/products?utm_source=mozilla.org&utm_medium=referral&utm_campaign=need-help-link) home page.
 
-The test can be done for all locales only on staging. For production testing, only these locales are enabled: de, en-GB, en-ZA, es-AR, es-CL, es-ES, es-MX, fr, id, pl, pt-BR, and ru.
-* Enter a mobile phone number, a new page will be generated, suggesting the next action to take.
-* Enter a phone number in the wrong phone number format. This will prompt an error message.
+On the right side of the page:
+* Click on the ````?```` at the end of **Which browser would you like to download?**, a popup window shows the descriptions of different versions of Firefox available for download.
+* Click on the ````?```` at the end of **Select your preferred installer**, a popup window shows the descriptions of different versions of installers available for download.
+* Under the section of **Select your preferred language**, check whether your current preferred language is shown by default. Change to a different language, check whether the download summary above the **Download Now** button corresponds to your new selection.
 
-### firefox/tracking-protection-tour.lang
+### [firefox/new/trailhead.lang](https://www.mozilla.org/firefox/new/)
 
-* Open Firefox and start a New Private Window.
-* Once the window is launched, click on **See how it works** button.
-* You are taken to [step one](https://www.mozilla.org/firefox/51.0.1/tracking-protection/start/?step=1).
-* Click the **Next** button, it takes you to steps [two](https://www.mozilla.org/firefox/51.0.1/tracking-protection/start/?step=2) and [three](https://www.mozilla.org/firefox/51.0.1/tracking-protection/start/?step=3).
-* In step [three](https://www.mozilla.org/firefox/51.0.1/tracking-protection/start/?step=3), an extra text box appears.
-* Click on **Disable protection for this session** button to generate a new “step three” message.
-* Click on **Enable protection** button to toggle between *enable* and *disable* protection messages.
-  * NOTE: in order to see the message of **Disable protection for this site**, the browser must be in _normal mode_.
-* Click on **Got it!** button to get to the next page.
-* Click on **Restart tour** button to go through the above steps.
+* Click the **Download Now** button to check the [thank you](https://www.mozilla.org/en-US/firefox/download/thanks) page that triggers an automatic download if not logged in to a Firefox account. If you have already logged in, a window will be popped up to invite you to save the updated Firefox download.
```

Comment:
Are you sure this is accurate? The download should start automatically independently if you're logged in to FxA or not.

Reply:
For me, when I am logged in using Fx browser, I get a "save the file" dialog box. And when I use a different browser (this way I am not logged in), it is automatically downloaded. Could you confirm? Or we can remove this part. It could be my setting in preference to prevent automatic download. That might be it!

Reply:
That has nothing to do with FxA, it's about your settings in that browser for downloads (automatic download, or ask where to save).